### PR TITLE
Script fixes

### PR DESCRIPTION
**Pull request by Margen67**
### Implementation
Read the commit(s).

### Checklist
- [x] This PR targets the `master` branch.
- [ ] This change has been filed as an issue.
- [x] No other PRs address this.
- [ ] This PR is as minimal as possible.
- [x] This PR does not introduce any regressions.
- [ ] This PR has been tested on all platforms, on both a dedicated server and on a listen server.

### Testing Checklist
|         |            Client             |            Server             | Version                     |
|---------|:-----------------------------:|:-----------------------------:|-----------------------------|
| Windows |  |  |     |
|   Linux |  |  |  |
|  Mac OS |  |  |       |

### CODE REVIEW

File: game_clean/copy/start_server.bat [hunk 1]
```diff
@@ -1,2 +0,0 @@
-@echo off
-srcds.bat +sv_pure 1 +maxplayers 32 +sv_lan 1 %* < nul
```
Comment (mastercoms) [inline game_clean/copy/start_server.bat:2]: start_server is separate, to isolate "required" launch flags with optional ones.

File: game_clean/copy/hl2.bat [hunk 2]
```diff
@@ -1,2 +1,2 @@
-@echo off
-@start hl2.exe -steam -game tf -insecure %*
+@ECHO OFF
+START hl2.exe -steam -insecure -novid -nojoy -nosteamcontroller -nohltv -nostartupsound %*
```
Comment (mastercoms) [inline game_clean/copy/hl2.bat:2]: This file is supposed to provide the set of "required" launch flags.

File: game_clean/copy/srcds.bat [hunk 3]
```diff
@@ -1,2 +1,2 @@
-@echo off
-srcds.exe -game tf -console -nomaster -insecure %*
+@ECHO OFF
+START srcds.exe -console -nomaster -insecure +sv_pure 1 +maxplayers 32 +sv_lan 1 %*
```
Comment (mastercoms) [inline game_clean/copy/srcds.bat:2]: This file is supposed to provide the set of "required" launch flags.

File: game_clean/copy/start_tf2.bat [hunk 4]
```diff
@@ -1,2 +1,2 @@
-@echo off
-hl2.bat -novid -nojoy -nosteamcontroller -nohltv -particle_fallback 2 %* < nul
+@ECHO OFF
+hl2.bat -game tf %* < nul
```
Comment (mastercoms) [inline game_clean/copy/start_tf2.bat:2]: start_tf2 is separate, to isolate "required" launch flags with optional ones.

File: game_clean/copy/start_tf2_server.bat [hunk 5]
```diff
@@ -0,0 +1,2 @@
+@ECHO OFF
+srcds.bat -game tf %* < nul
```
Comment (mastercoms) [inline game_clean/copy/start_tf2_server.bat:2]: Not sure what this file is for.

**Review by mastercoms — CHANGES_REQUESTED** [review]
Looks good, except for the launch scripts. I'd rather you revert the functionality changes on those.

File: game_clean/copy/hl2.bat (hunk 2 above)
Comment (Margen67) [reply game_clean/copy/hl2.bat:2]: Is `-particle_fallback 2` required?

Comment (mastercoms) [reply game_clean/copy/hl2.bat]: I don't think it was in this file, so no.

Comment (Margen67) [reply game_clean/copy/hl2.bat]: Oops. I meant in start_tf2.bat

File: game_clean/copy/start_tf2.bat (hunk 4 above)
Comment (Margen67) [reply game_clean/copy/start_tf2.bat]: Is `-particle_fallback 2` required? (right file this time lol)

Comment (mastercoms) [reply game_clean/copy/start_tf2.bat]: I added it there for convenience and so people can test the new option easily. It's not necessarily required but I would like to include it.

Comment (Margen67) [reply game_clean/copy/start_tf2.bat]: I'll add it back, then. Should I add `-nostartupsound`?

Comment (mastercoms) [reply game_clean/copy/start_tf2.bat]: No, I'd rather keep the current set of launch options with no changes.

File: game_clean/create_clean.sh [hunk 6]
```diff
@@ -1,26 +1,48 @@
...
-declare -a files=("../LICENSE_SDK" "../.github/README.md" "../thirdpartylegalnotices.txt")
-for F in "${files[@]}"; do
-  orig=$(basename $F)
-  cp -f $F $clean_folder/$orig
-done
+#!/bin/bash
+#
+# Run script within the directory
+BIN_DIR=$(dirname "$(readlink -fn "$0")")
+cd "${BIN_DIR}" || exit 2
```
Comment (Margen67) [inline game_clean/create_clean.sh:5]: @mastercoms One last thing: Does this need to return an error level of 2?

Comment (mastercoms) [reply game_clean/create_clean.sh:5]: Every failed action should have a unique error code, imo

**Comment by Margen67** [comment]
@mastercoms This should be ready to merge.

**Review by mastercoms — APPROVED** [review]

**Pull request merged**